### PR TITLE
code standard to follow

### PR DESCRIPTION
**Pull request by iamtekson**
This PR includes a code standard to follow within OSGeo Nepal repositories. Feel free to add or remove things if you have any.

### CODE REVIEW

**Comment by iamtekson** [comment]
This is work in progress for #9

**Comment by kshitijrajsharma** [comment]
@iamtekson  Let's reference original OSGeo community asks : https://wiki.osgeo.org/wiki/OSGeo_Community_Projects 
And also some examples like : https://live.osgeo.org/en/quickstart/osgeolive_quickstart.html 
This issue is mainly for setting standards for the projects we maintain or for the new project that comes for application 

OSGeo asks for the following : 
We can make this as a base and explain the steps in more detail ! 

image

File: Code_Standard.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Code Standard to follow
```
Comment (iamtekson) [inline Code_Standard.md:1]: For develpers

**Review by iamtekson — COMMENTED** [review]
Write more about community standards as shown in screenshot by @kshitijrajsharma

**Comment by iamtekson** [comment]
@kshitijrajsharma, For now, I just added the similar things from `OSGeo`. Things can be change later if we have other ideas.

**Review by kshitijrajsharma — APPROVED** [review]

**Pull request merged**